<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notifiacation</class>
 <widget class="QDialog" name="notifiacation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>25</number>
     </property>
     <item>
      <widget class="QLabel" name="notifications">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Notifications</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mute_all">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>25</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="zalo_icon">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>Zalo</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>D:/C/uet_3_2/tuong tac/library/zalo.png</normaloff>D:/C/uet_3_2/tuong tac/library/zalo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zalo_mute">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>25</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="mess_icon">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>Messenger</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>D:/C/uet_3_2/tuong tac/library/mess1.png</normaloff>D:/C/uet_3_2/tuong tac/library/mess1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mess_mute">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>24</number>
     </property>
     <item>
      <widget class="QPushButton" name="insta_icon">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>Instagram</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>D:/C/uet_3_2/tuong tac/library/instagram.png</normaloff>D:/C/uet_3_2/tuong tac/library/instagram.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="insta_mute">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>17</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="chorm_icon">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string>Chorm</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>D:/C/uet_3_2/tuong tac/library/chorm.png</normaloff>D:/C/uet_3_2/tuong tac/library/chorm.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="chorm_mute">
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</normaloff>D:/C/uet_3_2/tuong tac/library/bell.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
